index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Tesfaye-Portifolio-Project</title>
+  <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+  <div>
+    <header>
+    <nav>
+      <a href="index.html"><h3 class="logo pointer">TESFAYE.T</h3></a>
```

Comment:
- [ ] [OPTIONAL] Please see the below image and lookout for two red circles on the image. Please apply `text-decoration: none` on the `a` tag to remove the underlines. 

![image](https://user-images.githubusercontent.com/84156714/212697850-ae48ea24-4881-4247-98e4-e9cbeb1059ca.png)